Improve Run summary doc (#8434)
fishtown-analytics/docs.getdbt.com: website/docs/docs/deploy/run-visibility.md

### Run summary tab

> You can view and download in-progress and historical logs for your dbt runs. This makes it easier for the team to debug errors more efficiently. 

Reviewer: So we're not switching from second to third person
```suggestion
You can view and download in-progress and historical logs for your dbt runs. This makes it easier for you to debug errors more efficiently. 
```

> - To download the logs for each step, select the step in the **Run summary** tab and click **Download** > **Download logs**.

Reviewer: I know it's semantics, but to help it be a little clearer that this is logs for just that step and not historical data for any steps that came before it
```suggestion
- To download logs for an individual step, select the step in the **Run summary** tab and click **Download** > **Download logs**. 
```

> - Note that when viewing debug logs, the log output is truncated. To view and export all debug logs, click **Download** > **Download all debug logs**.

Reviewer: Same as above
```suggestion
- Note that when viewing debug logs, the log output is truncated. To view and export all debug logs for an individual step, click **Download** > **Download all debug logs**.
```

> <Lightbox src="/img/docs/dbt-cloud/deployment/download-logs.png" width="85%" title="Download logs" />

Reviewer 2: Thanks for adding this! Only concern is I think this reads like the debug truncation & download ability is in beta, which isn't true. All views have truncated debug logs & a download option, the button just moves with the new beta view.

Below is the GA'd view - what users will see if their run did not receive the beta view update. Debug logs are truncated & the button is inside of the log dropdown

<img width="1470" height="706" alt="Image" src="https://github.com/user-attachments/assets/40063cdf-819a-432d-a0fc-69e905b69a1f" />
